### ISSUE TITLE
Unskip amp-video and other video-related flaky tests

### ISSUE THREAD

**Issue by cathyxz** [ampproject/amphtml#16154]
Failed on master build: 
https://travis-ci.org/ampproject/amphtml/jobs/394220750
https://travis-ci.org/ampproject/amphtml/jobs/394225881

Stacktrace: 
```
DESCRIBE => amp-video
  DESCRIBE => Analytics Triggers
    IT => should trigger session analytics when a visible session ends
      ✗ Error: Timeout of 20000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```
Skipping this test for now. This is a tracking issue filed to investigate and unskip. 

Using this as a master issue to track all amp-video / video-player related tests that have been flaking on master. Skipping them for now. 
```
DESCRIBE => amp-dailymotion
  DESCRIBE =>  
    IT => renders responsively
      ✗ Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```

amp-video autoplay test
```
DESCRIBE => amp-video
  DESCRIBE => Autoplay
    DESCRIBE => play/pause
      IT => should play/pause when video enters/exits viewport
        ✗ Error: Timeout of 20000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```

**Comment by ampprojectbot** [comment]
This issue hasn't been  updated in awhile. @alanorozco Do you have any updates?

**Comment by aghassemi** [comment]
deleting unskip x issues. we can find them in the code
